src/utils/MerkleLib.sol

```diff
@@ -834,181 +836,185 @@ library MerkleLib {
     /**
      * @notice Calculates and returns the merkle root for the given leaf `_item`,
      * a merkle branch, and the index of `_item` in the tree.
-     * @param _item Merkle leaf
-     * @param _branch Merkle proof
-     * @param _index Index of `_item` in tree
-     * @return _current Calculated merkle root
+     * @param item Merkle leaf
+     * @param branch Merkle proof
+     * @param index Index of `_item` in tree
+     * @return current Calculated merkle root
      *
      */
-    function branchRoot(bytes32 _item, bytes32[TREE_DEPTH] memory _branch, uint256 _index)
+    function branchRoot(
+        bytes32 item,
+        bytes32[TREE_DEPTH] memory branch,
+        uint256 index
+    )
```

Comment:
Would be best to keep this library as close to the original, since any changes just increase the likelihood we introduce bugs.

Reply:
agreed if it aint broke...